ChangeLog.md

```diff
@@ -6,6 +6,14 @@
 
 * Fix breaking all empty annotations on merging the paragraph they are contained in with the one before ([#877](https://github.com/kogmbh/WebODF/pull/877)))
 
+### Improvements
+
+* In OpAddAnnotation support annotated ranges with 0 length (WARNING: NON-BACKWARD COMPATIBLE spec change) ([#879](https://github.com/kogmbh/WebODF/pull/879)))
```

Comment:
Never heard before, thanks for the hint. Suspect that is MS lingo, so that could be why ;)
Still, seems it is quite in use and also a short term, so happy to use it.
Hm, wondering if that should be a separate section even, and not just a inline comment. Guess it should.